Libraries/LibWeb/ARIA/Roles.h

```diff
@@ -39,6 +39,9 @@ namespace Web::ARIA {
     __ENUMERATE_ARIA_ROLE(figure)           \
     __ENUMERATE_ARIA_ROLE(form)             \
     __ENUMERATE_ARIA_ROLE(generic)          \
+    __ENUMERATE_ARIA_ROLE(graphicsdocument) \
+    __ENUMERATE_ARIA_ROLE(graphicsobject)   \
+    __ENUMERATE_ARIA_ROLE(graphicssymbol)   \
```

Comment:
See Libraries/LibWeb/ARIA/AttributeNames.h for an example on how to handle hyphens in xmacro values. Basically, we just do this:
```c++
__ENUMERATE_ARIA_ROLE(graphicsdocument, "graphics-document")
```

And then the call site becomes something like:
```c++
Optional<Role> role_from_string(StringView role_name)
{
#define __ENUMERATE_ARIA_ROLE(name, attribute)                       \
    if (role_name.equals_ignoring_ascii_case(attribute##sv))         \
        return Role::name;
    ENUMERATE_ARIA_ROLES
#undef __ENUMERATE_ARIA_ROLE
    return {};
}
```

Reply:
> See Libraries/LibWeb/ARIA/AttributeNames.h for an example on how to handle hyphens in xmacro values.

🎉 Beautiful — thanks much